packages/next/client/index.js

```diff
@@ -73,7 +74,12 @@ export default async ({
   if (process.env.NODE_ENV === 'development') {
     webpackHMR = passedWebpackHMR
   }
-  App = await pageLoader.loadPage('/_app')
+
+  try {
```

Comment:
This will bundle both files. Instead we should alias the file in the webpack config, should be relatively easy by just passing the path from the `pages` in `build/index.ts`

```
const pages = createPagesMapping(pagePaths, config.pageExtensions)
```

This will hold `pages['/_app']` with the absolute path